api/models/user_activities.py

```diff
@@ -1,18 +1,49 @@
+from datetime import date
+from flask_sqlalchemy.query import Query as BaseQuery
+from sqlalchemy import desc, func
+
 from api import db
 from api.mixins import TimestampMixin
 
 
+class UserActivitiesQuery(BaseQuery):
+    def get_started_activity_for_user(self, user_id, user_phone):
```

Comment:
shouldn't it be like this?
```suggestion
    def get_todays_started_activity_for_user(self, user_id, user_phone):
```